+++ /usr/local/bundle/gems/middleman-search-gds-0.11.1/lib/middleman-search/search-index-resource.rb
@@ -111,6 +111,8 @@
           end
         end

+        store.each { |id, item| item[:url].sub!(/^/, ENV["ROOT_DOCPATH"].to_s) }
+
         # Generate JSON output
         context = ExecJS.compile(source.join("\n"))
         json = context.eval('lunr.middlemanSearchIndex.indexJson()')
